openff/interchange/components/potentials.py

```diff
@@ -25,15 +26,13 @@
 )
 from openff.interchange.pydantic import _BaseModel
 
-if has_package("jax"):
-    from jax import numpy as jax_numpy
-
-from numpy.typing import ArrayLike
-
-if has_package("jax"):
-    from jax import Array
+if TYPE_CHECKING:
+    if has_package("jax"):
+        from jax import Array
+    else:
+        Array = Any  # type: ignore
```

Comment:
This [seems](https://docs.python.org/3/library/typing.html#type-aliases) to be the preferred syntax for type aliases prior to 3.12 - does it resolve the `type:ignore`?

```suggestion
        Array: TypeAlias = Any
```